releasenotes/notes/target-aware-layout-routing-2b39bd87a9f928e7.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    The following layout and routing transpiler passes from the
+    :mod:`.qiskit.transpiler.passes` modules have a new keyword argument,
+    ``target`` which takes in a :class:`~.Target` object which is used to model
+    the constraints of a target backend. If the ``target`` keyword argument is
+    specified it will be used as the source of truth for any hardware
+    constraints used in the operation of the transpiler pass. It will
+    superscede any other arguments for specifying hardware constraints,
```

Comment:
supersede (no c)

Reply:
I will learn how to spell this word one day...